packages/fxa-content-server/app/styles/modules/_settings.scss

```diff
@@ -321,7 +321,12 @@ body.settings #stage .settings {
 .settings-unit-details {
   clear: both;
   display: none;
-  padding: 0 16px;
+  @include respond-to('big') {
```

Comment:
r+ for the code. But I guess you have to add a newline before `@include respond-to('big') {` so that the code goes with the rest.

Reply:
Done, thanks @hritvi!